supereeg/simulate.py

```diff
@@ -15,12 +15,22 @@ def simulate_locations(n_elecs=10):
     n_elecs : int
         Number of electrodes
 
+    random_seed : bool or int
+        Default False.  If True, set random seed to 123.  If int, set random
+        seed to value.
+
     Returns
     ----------
     elecs : pd.DataFrame
         A location by coordinate (x,y,z) matrix of simulated electrode locations
 
     """
+    if random_seed:
+        if isinstance(random_seed, int):
+            np.random.seed(random_seed)
```

Comment:
I think we want an additional case where if `not random_seed` we call `np.random.seed(None)` to pick one randomly...

Reply:
Sorry...this should actually do something like:
- if random_seed is True, we want `np.random.seed(None)`
- elif random_seed is an int, we want `np.random.seed(random_seed)`
- else we want `np.random.seed(123)`